plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -239,6 +242,38 @@ export class KubernetesBuilder {
       }
     });
 
+    router.post('/resources/workloads', async (req, res) => {
+      const requestBody: ObjectsByEntityRequest = req.body;
+      const entityName = requestBody.entity.metadata.name;
```

Comment:
Best avoid introducing any more handling of entity name alone tbh, rather use `stringifyEntityRef` to show the full ref or if you need to pass things around as a string

Reply:
done